scripts/copy-build-to-dist.mjs

```diff
@@ -0,0 +1,88 @@
+import path from "path";
+import fse from "fs-extra";
+import chalk from "chalk";
+
+const ROOT_DIR = process.cwd();
+const BUILD_PATH = path.join(ROOT_DIR, "build", "node_modules");
```

Comment:
I don't think it's anything we. need to deal with, but this doesn't take into account `REMIX_LOCAL_DEV_OUTPUT_DIRECTORY` which makes for an interesting flow:

* Assume no existing `build/` directory
* `REMIX_LOCAL_DEV_OUTPUT_DIRECTORY=../some-app yarn build`
* `rollup` builds output to the specific app folder
* `tsc` doesn't have awareness of that env var (a known current issue), so it still builds types to `build/`
* This script then copies only the updates types to `dist/` leaving them potentially out of sync with the source code in `dist/`

If the rollup ts plugin wasn't so slow I think this would be a non-issue.   And I think it's also fine to ignore since it's just a local dev shortcut and we can just work around it.  Just wanted to call it out for visibility though

Reply:
I went ahead and updated this to take that env variable into account. Better to copy just the js files than just the types! Hopefully we can figure out a decent solution for types later.